sdk/cognitivelanguage/Azure.AI.Language.Conversations/samples/Sample6_AnalyzeConversation_ConversationSummarization.md

```diff
@@ -26,104 +26,77 @@ Once you have created a client, you can call synchronous or asynchronous methods
 ## Synchronous
 
 ```C# Snippet:AnalyzeConversation_ConversationSummarization
-var data = new
-{
-    AnalysisInput = new
-    {
-        Conversations = new[]
+AnalyzeConversationOperationInput data = new AnalyzeConversationOperationInput(
+    new MultiLanguageConversationInput(
+        new List<ConversationInput>
         {
-            new
+            new TextConversation("1", "en", new List<TextConversationItem>()
             {
-                ConversationItems = new[]
+                new TextConversationItem("1", "Agent_1", "Hello, how can I help you?")
                 {
-                    new
-                    {
-                        Text = "Hello, how can I help you?",
-                        Id = "1",
-                        Role = "Agent",
-                        ParticipantId = "Agent_1",
-                    },
-                    new
-                    {
-                        Text = "How to upgrade Office? I am getting error messages the whole day.",
-                        Id = "2",
-                        Role = "Customer",
-                        ParticipantId = "Customer_1",
-                    },
-                    new
-                    {
-                        Text = "Press the upgrade button please. Then sign in and follow the instructions.",
-                        Id = "3",
-                        Role = "Agent",
-                        ParticipantId = "Agent_1",
-                    },
+                    Role = ParticipantRole.Agent
                 },
-                Id = "1",
-                Language = "en",
-                Modality = "text",
-            },
-        }
-    },
-    Tasks = new[]
-    {
-        new
-        {
-            TaskName = "Issue task",
-            Kind = "ConversationalSummarizationTask",
-            Parameters = new
-            {
-                SummaryAspects = new[]
+                new TextConversationItem("2", "Customer_1", "How to upgrade Office? I am getting error messages the whole day.")
```

Comment:
I still see this without the named variables on my side

Reply:
Fixed in the newest iteration : https://github.com/Azure/azure-sdk-for-net/blob/5ea159c6046674d12aaa903e1e3ab98edf636ebd/sdk/cognitivelanguage/Azure.AI.Language.Conversations/samples/Sample6_AnalyzeConversation_ConversationSummarization.md